packages/dm-core-plugins/src/explorer/style.css

```diff
@@ -0,0 +1,3 @@
+body .lm_content {
+  overflow: scroll;
```

Comment:
Same goes for this one right?
I should have pointed out both right away

Reply:
ups, forgot to push that change. Will fix!